schemas/test/en/test_introduction.json

```diff
@@ -707,7 +707,7 @@
                                 "title": "Section complete",
                                 "contents": [
                                     {
-                                        "description": "<p>You have successfully completed this section</p><p>The next section covers changes in <strong>business strategy and practices</strong>, for example, implementing changes to marketing concepts or strategies.</p>"
+                                        "description": "You have successfully completed this section. The next section covers changes in <strong>business strategy and practices</strong>, for example, implementing changes to marketing concepts or strategies."
```

Comment:
Shouldn't this be a list of two elements? Like you the one you've added in `test_benchmark_business.json`?

Reply:
Updated 👍